src/api.ts

```diff
@@ -304,22 +304,23 @@ router.post(
       // Once confirmed, move to archive collection
       await database.cancelBid(bidData, archiveCollection);
 
-      // const message: TypedMessage<BidCancelledV1Data> = {
-      //   event: MessageType.BidCancelled,
-      //   version: "1.0",
-      //   timestamp: new Date().getTime(),
-      //   logIndex: undefined,
-      //   blockNumber: undefined,
-      //   data: {
-      //     account: signer,
-      //     uniqueBidId: bidData.uniqueBidId,
-      //   },
-      // };
+      const message: TypedMessage<BidCancelledV1Data> = {
+        event: MessageType.BidCancelled,
+        version: "1.0",
+        timestamp: new Date().getTime(),
+        logIndex: undefined,
+        blockNumber: undefined,
+        data: {
+          account: signer,
+          uniqueBidId: bidData.uniqueBidId,
+        },
+      };
 
-      // await queue.sendMessage(message);
+      await queue.sendMessage(message);
+      // await queue.close(); 
```

Comment:
where is the connection being opening/closed now? (also, remove if unneeded)

Reply:
The connection is both opened and closed every time you send a message now. Weird these comments are here, they don't show on my local at this commit